data/api/client-server/definitions/sync_filter.yaml

```
@@ -16,10 +16,9 @@ title: Filter
 properties:
   event_fields:
     description: List of event fields to include. If this list is absent then all
-      fields are included. The entries may include '.' characters to indicate sub-fields.
-      So ['content.body'] will include the 'body' field of the 'content' object. A
-      literal '.' character in a field name may be escaped using a '\\'. A server may
-      include more fields than were requested.
+      fields are included. The entries are [dot-separated paths for each property](/appendices#dot-separated property paths)
```

This markdown link doesn't appear to work. I think you might need to include some more hyphens?

```suggestion
      fields are included. The entries are [dot-separated paths for each property](/appendices#dot-separated-property-paths)
```

Likewise for the other places the Appendices is referenced in this PR.

---

Oops. 👍